### PR TITLE
basic auth strategy

### PR DESCRIPTION
**Pull request by comensee**

### CODE REVIEW

File: src/RestRemoteObject/Client/Rest/Authentication/BasicAuthAuthenticationStrategy.php [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+        return $this->user;
+    }
+
+    /**
+     * Set the username
+     * @param $user
+     * @return $this
+     */
+    public function setUser($user)
```
Comment (Ocramius) [inline src/RestRemoteObject/Client/Rest/Authentication/BasicAuthAuthenticationStrategy.php:65]: Not needed

File: src/RestRemoteObject/Client/Rest/Authentication/BasicAuthAuthenticationStrategy.php [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+        return $this->password;
+    }
+
+    /**
+     * Set the password
+     * @param $password
+     * @return $this
+     */
+    public function setPassword($password)
```
Comment (Ocramius) [inline src/RestRemoteObject/Client/Rest/Authentication/BasicAuthAuthenticationStrategy.php:91]: Not needed

File: src/RestRemoteObject/Client/Rest/Authentication/BasicAuthAuthenticationStrategy.php [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+        return $this;
+    }
+
+    /**
+     * Get the password
+     * @return string
+     * @throws \RestRemoteObject\Client\Rest\Exception\MissingAuthenticationParameterException
+     */
+    public function getPassword()
```
Comment (Ocramius) [inline src/RestRemoteObject/Client/Rest/Authentication/BasicAuthAuthenticationStrategy.php:77]: Not needed

File: src/RestRemoteObject/Client/Rest/Authentication/BasicAuthAuthenticationStrategy.php [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+        $this->client->setAuth($user, $password);
+    }
+
+    /**
+     * Get the username
+     * @return string
+     * @throws \RestRemoteObject\Client\Rest\Exception\MissingAuthenticationParameterException
+     */
+    public function getUser()
```
Comment (Ocramius) [inline src/RestRemoteObject/Client/Rest/Authentication/BasicAuthAuthenticationStrategy.php:51]: Not needed

**Comment by Ocramius** [comment]
Isn't this file already available at https://github.com/fourmation/RestRemoteObject/blob/8fca4c9bbe1ea6c2eb075230abb8b608349310ac/src/RestRemoteObject/Client/Rest/Authentication/HttpAuthenticationStrategy.php ?

**Comment by comensee** [comment]
Sorry for disturbing guys,
We decided we will do the modification on our code.

**Comment by dkcwd** [comment]
No worries Alain, the repo has been quiet for a while ;-)

**Comment by blanchonvincent** [comment]
quiet because there is no bugs or quiet because the project is not popular? :p

**Comment by dkcwd** [comment]
:-D no bugs of course!

**Pull request closed without merging**